.flake8/azul_flake8.py

```diff
@@ -295,11 +329,15 @@ class AzulImports:
     name = 'azul_imports'
     version = 1.0
 
-    def __init__(self, tree, file_tokens):
+    def __init__(self, tree, file_tokens, filename):
```

Comment:
```suggestion
    def __init__(self, tree, file_tokens, file_name):
```

Reply:
I cannot change this parameter because flake8 decides the names of the keyword args passed to the plugin. Changing the parameter to `file_name` raises a `PluginRequestedUnknownParameters` exception.

Reply:
```suggestion
    # The constructor signature is subject to the restrictions documented at 
    # https://flake8.pycqa.org/en/3.8.2/plugin-development/plugin-parameters.html#indicating-desired-data
    def __init__(self, *, tree, file_tokens, filename):
```

Reply:
If the * doesn't work, leave it off.